src/pyramid/config/routes.py

```diff
@@ -335,9 +335,10 @@ def add_route(
         if 'effective_principals' in predicates:
             warnings.warn(
                 (
-                    'The new security policy has removed the concept of '
-                    'principals. See "Upgrading Authentication/Authorization" '
-                    'in "What\'s New in Pyramid 2.0" for more information.'
+                    'The new security policy has deprecated '
+                    'effective_principals. See "Upgrading '
+                    'Authentication/Authorization" in "What\'s New in '
+                    'Pyramid 2.0" for more information.'
```

Comment:
```suggestion
                    'Pyramid 2.0" in the Pyramid documentation for more information.'
```

It might not be obvious for the user to look in the documentation. Update elsewhere, too.